bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,181 @@
+from asyncio import TimeoutError
+from pathlib import Path
+from random import choice
+from typing import Literal
+
+from discord import Embed, Message
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+# Defining all words in the list of words as a global variable
+with Path("bot/resources/fun/hangman_words.txt").resolve().open(mode="r", encoding="utf-8") as f:
```

Comment:
```suggestion
with open("bot/resources/fun/hangman_words.txt", encoding="utf-8") as f:
```
Wouldn't this be easier to read?
I'm not sure if there's a reason you're passing it through pathlib and then resolving it. Let me know if there is. Removing this means you can also remove the Path import.